subjects/graphql/index.html

```diff
@@ -1,119 +1,37 @@
-  font-family: sans-serif;
-  letter-spacing: 1.5px;
-  display: flex;
-  flex-wrap: wrap;
-  justify-content: center;
-  align-content: center;
-  width: inherit;
-  height: inherit;
-  padding: 5rem;
-  font-size: 12px;
-}
-
-input {
-  border: none;
-  position: absolute;
-  margin: 6px 24px 8px 15px;
-  background: rgba(255, 255, 255, 0);
-  border-bottom: 1px solid var(--color);
-  outline: none;
-}
-
-.docExplorerWrap {
-  position: inherit;
-  width: 80%;
-  height: 100%;
-  top: 50%;
-  left: 50%;
-  background-color: hsl(0, 1%, 35%);
-}
-
-.doc-title, .doc-explore-content {
-  padding: 10px;
-  margin: 10px;
-  align-content: center;
-  border-bottom: 1px solid var(--color);
-}
-
-.doc-title {
-  display: flex;
-}
-
-.back {
-  width: 50px;
-  height: 50px;
-}
-
-h3, h4, li {
-  font-weight: 300;
-  width: 100%;
-  margin: 15px;
-}
-
-h3 {
-  font-size: 1.25rem;
-}
-
-h4 {
-  font-size: 0.8rem;
-  white-space: pre;
-}
-
-b {
-  padding: 4px;
-  border-bottom: 1px solid var(--color);
-}
-
-a {
-  color:lightcoral;
-}
-
-a:hover {
-  text-decoration: underline;
-  cursor: pointer;
-}
-
-img {
-  cursor: pointer;
-  width: 30px;
-  height: 30px;
-}
-
-</style>
-</head>
-<body>
-  <script type="module" src="./lib/schema.js"></script>
-  <div class="docExplorerWrap">
-      <div class="doc-title"><h3 class="title">Schema Documentation</h3></div>
-      <div class="doc-explore-content">
-        <label class="search-box">
-          <img src="https://cdn1.iconfinder.com/data/icons/thin-ui-1/100/Noun_Project_100Icon_1px_grid_thin_ic_cookie-512.png">
-          <input class="search" type="text" placeholder="Search Schema...">
-        </label>
-        <div class="doc-type-description"> <h4 class="comment">A Graphql schema provides a root type for each kind of operation</h4><!-- description of... --></div>
-        <br>
-        <div class="doc-category"><h4><b class="doc-type">ROOT TYPES</b></h4></div>
-        <br>
-        <div class="display-doc"></h4></div>
-      </div>
-  </div>
-</body>
-</html>
+  <head>
+    <title>Simple GraphiQL Example</title>
+    <link href="https://unpkg.com/graphiql/graphiql.min.css" rel="stylesheet" />
+  </head>
+  <body style="margin: 0;">
+    <div id="graphiql" style="height: 100vh;"></div>
+
+    <script
+      crossorigin
+      src="https://unpkg.com/react/umd/react.production.min.js"
+    ></script>
+    <script
+      crossorigin
+      src="https://unpkg.com/react-dom/umd/react-dom.production.min.js"
+    ></script>
+    <script
+      crossorigin
+      src="https://unpkg.com/graphiql/graphiql.min.js"
+    ></script>
+
+    <script>
+      const graphQLFetcher = graphQLParams =>
+        fetch(`https://${location.hostname}/api/graphql-engine/v1/graphql`, {
+          method: 'post',
+          headers: { 'Content-Type': 'application/json' },
+          body: JSON.stringify(graphQLParams),
+        })
+          .then(response => response.json())
+          .catch(() => response.text());
```

Comment:
```js
   .catch(() => response.text())
```
wat ?

Here `response` is not defined, also you don't do  anything with the error.

I believe it's best not to catch at all and let `graphiQL` handle the error.
> edit: Ok i've check graphiql examples, we should do:
> ```js
> .then(response => response.json().catch(() => response.text()))
> ```

It's only if the `.json()` fail that we fallback to text

Reply:
ok ok will correct it, thanks